### DISCUSSION TITLE
How does storybook proxy the request to the api route of next.js

### DISCUSSION THREAD

**Discussion by yjhtry** [storybookjs/storybook#21707] [Help]
When I write a story for an upload component, I can't call the api routes of next.js. I try to use webpack proxy configuration, but it doesn't work

```
    "storybook": "^7.0.0-rc.5",
```

This is my configuration
```javascript
{
  webpackFinal: async (config) => ({
    ...config,
    resolve: {
      ...config.resolve,
      alias: {
        ...config.resolve?.alias,
        '@': path.resolve(__dirname, '../'),
      },
    },
    devServer: {
      ...config.devServer,
      proxy: {
        '/api': 'http://localhost:3000',
      },
    },
  })
}
```
This is an error message
<img width="538" alt="image" src="https://user-images.githubusercontent.com/39440537/226551525-76a7d825-0e59-4e13-94d2-e5f445b512a7.png">

Port 3000 works properly
<img width="363" alt="image" src="https://user-images.githubusercontent.com/39440537/226551703-253dd341-c145-401a-9745-f9adce4f8e3b.png">

**Comment by rcogal** [comment]
@yjhtry are you able to resolve your problem? I am still having an issue on this

**Reply by rcogal** [reply]
I just thought it might also solve your problem. Found an alternative solution for setting proxy

https://github.com/storybookjs/storybook/issues/208
